src/browser/app/Large.styl

```diff
@@ -0,0 +1,181 @@
+// Main
+@media screen and (min-width: 1620px)
+  .screen
+
+    .screen-content
+      .item
+        font-size 1.2vw
+
+      .options
+        .option
+          p
+            font-size 1.4vw
+          .icon
+            width 3.2vw
+            height 3.2vw
+
+      h3
+        font-size 1.2vw
+
+    .content
+      .screen-desc
+        font-size 1.4vw
+
+      h1
+        font-size 2.5vw
+        text-transform uppercase
+
+      ul
+        li
+          font-size 1.2vw
+          line-height 2.5vw
+          margin-left 1.6vw
+        li:before
+          width 1.6vw
+          margin 2px 5px 0 -2vw
+
+    .chevron
+      i
+        width 1.5vw
+        height 1.5vw
+        border-width 10px
+      i.-red
+        border-width 10px
+
+  .hello
+    font-size 1vw
+
+  // First
+  .screen.first-screen
+    .centering-wrapper
+      h1
+        font-size 4vw
+      h2
+        font-size 1.6vw
+
+      .form-control
+        width 24vw
+        > input
+          font-size 1.6vw
+
+  // Second
+  .screen.second-screen
+    h2
+      font-size 1.6vw
+
+    > div
+      .right-side
+        .path
+          .scalac
+            width 2.5vw
+
+  // Roles
+  .screen.roles-screen
+    h1
+      font-size 2.5vw
+    ul
+      li
+        div
+          p
+            font-size 1.7vw
+
+  // Position
+
+  // Compass
+  .screen.compass-screen
+    .values
+      .item
+        img
+          height 3.5vw
+        span
+          font-size 1vw
+    .rules
+      img
+        height 4vw
+
+  .job
+    h3
+      font-size 1.4vw
+
+  // Equipment
+
+  // Skills
+  .screen.skills-screen
+    h1
+      font-size 2.5vw
+    .right
+      .screen-title
+        p
+          font-size 1.4vw
+    ul
+      li
+        width 33.3%
+        img
+          max-width 4vw
+
+    .skills-block
+      .tags-wrapper
+        p
+          font-size 1.2vw
+      .react-tagsinput-input
+        font-size 1.1vw
+        width inherit
+
+  // Check
+  .screen.check-screen
+    .checkbox-list
+      margin-left 2.5vw
+    .list-item
+      margin .7vw 0
+    label
+      font-size 1.1vw
+    label:before
+      width 1.6vw
+      height 1.6vw
+      margin-left -2.5vw
+
+  // Experience
+  .screen.experience-screen
+    .experience
+      p
+        font-size 1.2vw
+      .note
+        p
+          font-size 1vw
+
+    .experience-level
+
+      .level
+        height 16vw
+        h4
+          font-size 1.2vw
+        h5
+          font-size 1vw
+        img
+          width 3vw
+
+  // Submit
+  .screen.submit
+    header
+      font-size 2.5vw
+    h2
+      max-width 30vw
+      font-size 1.6vw
+    input[type=text],
+    input[type=url],
+    input[type=email]
+      width 30vw
+      font-size 1.4vw
+    #drop
+      width 30vw
+    span
+      font-size 1.4vw
+    button
+      width 30vw
+      height 5vw
+      i
+        width 2vw
+        height 2vw
+
+    .field
+      margin 1vw 0
```

Comment:
Brak nowej lini na koncu. Dobra praktyka wstawic w kazdym pliku nowa linie. Niektore edytory maja problem z czytaniem pliku jak nie ma nowej lini na koncu. Mozna w ustawieniach edytora zaznaczyc opcje, zeby robil to automatycznie. Polecam, nie trzeba potem pamietac o tym ;)